src/System.Management.Automation/resources/ParserStrings.resx

```diff
@@ -1355,7 +1355,7 @@ ModuleVersion : Version of module to import. If used, ModuleName must represent
 '{1}'.</value>
   </data>
   <data name="CantActivateDocumentInPowerShellCore" xml:space="preserve">
-    <value>Cannot run a document in PowerShell Core: {0}.</value>
+    <value>Cannot run a document in PowerShell: {0}.</value>
```

Comment:
I thought we fixed this so this error should no longer come up?  If it's still applicable, put back Core

Reply:
I don't know about applicable but `CantActivateDocumentInPowerShellCore` is still used in `NativeCommandProcessor.cs ` for Nano and IoT.

Reverted.